addons/web/static/src/js/report/client_action.js

```diff
@@ -42,14 +42,14 @@ var ReportAction = AbstractAction.extend({
         this.controlPanelProps.cp_content = {
             $buttons: this.$buttons,
         };
-        return Promise.all([this._super.apply(this, arguments), session.is_bound]).then(async function () {
+        return Promise.all([this._super.apply(this, arguments), session.is_bound]).then(async () => {
```

Comment:
why is this file moved into static/src/js/report?

Reply:
Because it is a client action specifically for reports, so it goes inside the report dir